packages/server-boilerplate/src/orchestrator/auth/AuthConnection.ts

```diff
@@ -31,6 +24,13 @@ export class AuthConnection extends ApiConnection {
   public baseUrl = process.env.CENTRAL_API_URL || 'http://localhost:8090/v2'; // auth server is actually just central server
 
   public constructor() {
+    const basicAuthHandler = {
+      getAuthHeader: async () => {
+        const API_CLIENT_NAME = requireEnv('API_CLIENT_NAME');
+        const API_CLIENT_PASSWORD = requireEnv('API_CLIENT_PASSWORD');
+        return createBasicHeader(API_CLIENT_NAME, API_CLIENT_PASSWORD);
+      },
+    };
```

Comment:
In this case we can leave it at the root level, since it's wrapped in a function, so `requireEnv` will get called at the time the function gets called :+1: